### PR TITLE
Fix pac-secret-reaper CronJob

### PR DESCRIPTION
**Pull request by Roming22**
Fix the query that retrieves the list of old secrets

### CODE REVIEW

File: operator/gitops/argocd/pipeline-service/openshift-pipelines/bugfix-pac-gitauth-secrets.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ spec:
                       jq -r '.items[] |
                         select(.metadata.name |
                         startswith("pac-gitauth-")) |
-                        select(.metadata.creationTimestamp < (now - 86400)) |
+                        select(.metadata.creationTimestamp | fromdateiso8601 < (now - 86400)) |
```
[the remark this answers is not in the document]
Comment (Roming22) [reply operator/gitops/argocd/pipeline-service/openshift-pipelines/bugfix-pac-gitauth-secrets.yaml:114]: I would be surprised if  I could create a resource and specify its creationTimestamp to be in the past.

The other option I see would be to put that into a script with params to control the "older than" value, which could then be tested.
But that means having to build/deliver a new image. That seems overkill in this situation.

[the remark this answers is not in the document]
Comment (gabemontero) [reply operator/gitops/argocd/pipeline-service/openshift-pipelines/bugfix-pac-gitauth-secrets.yaml:114]: so be it - standard question in these type of cases

if this was written in golang vs. bash, a unit test would have been straight forward ;-)

**Review by gabemontero — APPROVED** [review]

**Pull request merged**